app/views/paintings/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= simple_form_for @painting do |f| %>
+  <%= f.input :title %>
+  <%= f.button :submit %>
+<% end %>
```

Comment:
I think we need to add all the other columns that Painting can have. (check the schema file)